src/guides/install/local-installation.md

```diff
@@ -67,16 +67,12 @@ Then, update each individual program, as necessary.
 ```sh
 conda activate nextstrain
 
-# Update Nextstrain dependencies (mafft, etc.).
+# Update Nextstrain dependencies (mafft, etc.), Augur, and Nextstrain CLI.
 conda update --all
 
-# Update Augur.
-python3 -m pip install --upgrade nextstrain-augur
-
 # Update Auspice.
 npm update --global auspice
 
-# Update the Nextstrain CLI and pull the latest Docker image.
-python3 -m pip install --upgrade nextstrain-cli
+# Pull the latest Docker image for the CLI.
```

Comment:
One shouldn't need to upate the docker image for a "local" install, but I wonder how hard it would be to have `nextstrain update` also update itself and (conda-installed) augur/auspice?

Reply:
That's a great catch! I didn't notice that inconsistency on the last round of edits. We could move this to the CLI install docs which don't mention the `nextstrain update` command.

I like the idea of having `nextstrin update` update the Docker or conda environments, as needed. @tsibley mentioned something like this would be possible in the existing CLI framework.

Reply:
Yep! The middle-distance plan is to have `nextstrain update` be able to both update itself and a Conda-based environment.